test/dash-playlist-loader.test.js

```diff
@@ -16,8 +16,8 @@ import {
 } from './test-helpers';
 // needed for plugin registration
 import '../src/videojs-http-streaming';
-import testDataManifests from './dist/test-manifests.js';
-import { sidx as sidxResponse } from './dist/test-segments';
+import testDataManifests from 'create-test-data!manifests';
```

Comment:
anything that uses manifests or segments is now `create-test-data!segments/manifests` so that it can interface with the plugin.